dinersclub/reloadly.go

```diff
@@ -74,6 +59,54 @@ func reloadlyErrorResult(res *Result, err error, details *json.RawMessage) (*Res
 	return res, err
 }
 
+func (p *ReloadlyProvider) GetUserFromPaymentEvent(event *PaymentEvent) (*User, error) {
```

Comment:
In its current implementation, there's nothing provider-specific about this function. While "page" is not a very permanent concept once we move beyond Facebook, I think there will always be some uniform entityid on all events. What are your thoughts here?